code/game/objects/items/pneumaticCannon.dm

```diff
@@ -93,7 +93,13 @@
 	if(needs_air == FALSE)
 		return
 	playsound(src, 'sound/items/ratchet.ogg', 50, TRUE)
-	pressureSetting = pressureSetting >= HIGH_PRESSURE ? LOW_PRESSURE : pressureSetting + 1
+	switch(pressureSetting)
```

Comment:
I'm skeptical you can just change these numbers and nothing else. Are you sure they are not treated as a number anywhere?

Reply:
No idea how to do it then.

Reply:
Make a proc that returns the current pressure setting as a string

Reply:
That'll do?